packages/api/models/nomination.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  
+  class Nomination extends Model {};
+
+  Nomination.init({
+    id: DataTypes.UUIDV4,
+    status: DataTypes.STRING,
```

Comment:
Is it worth adding the `allowNull: false` constraint here, as well? I can see it going either way, but may be useful if we want to do any validation.